### ISSUE TITLE
Make CLI options as compatible with geth as possible

### ISSUE THREAD

**Issue by gavofyork** [ethcore/parity#572]

**Comment by arkpar** [comment]
```
NAME:
   geth - the go-ethereum command line interface

USAGE:
   geth [options] command [command options] [arguments...]

VERSION:
   1.4.0-unstable

COMMANDS:
   import	import a blockchain file
   export	export blockchain into file
   upgradedb	upgrade chainblock database
   removedb	Remove blockchain and state databases
   dump		dump a specific block from storage
   monitor	Geth Monitor: node metrics monitoring and visualization
   makedag	generate ethash dag (for testing)
   gpuinfo	gpuinfo
   gpubench	benchmark GPU
   version	print ethereum version numbers
   wallet	ethereum presale wallet
   account	manage accounts
   console	Geth Console: interactive JavaScript environment
   attach	Geth Console: interactive JavaScript environment (connect to node)
   js		executes the given JavaScript files in the Geth JavaScript VM
   help, h	Shows a list of commands or help for one command

ETHEREUM OPTIONS:
  --datadir "/Users/arkady/Library/Ethereum"	Data directory for the databases and keystore
  --networkid "1"				Network identifier (integer, 0=Olympic, 1=Frontier, 2=Morden)
  --olympic					Olympic network: pre-configured pre-release test network
  --testnet					Morden network: pre-configured test network with modified starting nonces (replay protection)
  --dev						Developer mode: pre-configured private network with several debugging flags
  --genesis 					Insert/overwrite the genesis block (JSON format)
  --identity 					Custom node name
  --fast					Enable fast syncing through state downloads
  --lightkdf					Reduce key-derivation RAM & CPU usage at some expense of KDF strength
  --cache "0"					Megabytes of memory allocated to internal caching (min 16MB / database forced)
  --blockchainversion "3"			Blockchain version (integer)

ACCOUNT OPTIONS:
  --unlock 	Comma separated list of accounts to unlock
  --password 	Password file to use for non-inteactive password input

API AND CONSOLE OPTIONS:
  --rpc								Enable the HTTP-RPC server
  --rpcaddr "localhost"						HTTP-RPC server listening interface
  --rpcport "8545"						HTTP-RPC server listening port
  --rpcapi "eth,net,web3"					API's offered over the HTTP-RPC interface
  --ws								Enable the WS-RPC server
  --wsaddr "localhost"						WS-RPC server listening interface
  --wsport "8546"						WS-RPC server listening port
  --wsapi "eth,net,web3"					API's offered over the WS-RPC interface
  --wsdomains 							Domains from which to accept websockets requests (can be spoofed)
  --ipcdisable							Disable the IPC-RPC server
  --ipcapi "admin,eth,debug,miner,net,shh,txpool,personal,web3"	API's offered over the IPC-RPC interface
  --ipcpath "geth.ipc"						Filename for IPC socket/pipe within the datadir (explicit paths escape it)
  --rpccorsdomain 						Domains from which to accept cross origin requests (browser enforced)
  --jspath "."							JavaScript root path for `loadScript` and document root for `admin.httpGet`
  --exec 							Execute JavaScript statement (only in combination with console/attach)

NETWORKING OPTIONS:
  --bootnodes 		Comma separated enode URLs for P2P discovery bootstrap
  --port "30303"	Network listening port
  --maxpeers "25"	Maximum number of network peers (network disabled if set to 0)
  --maxpendpeers "0"	Maximum number of pending connection attempts (defaults used if set to 0)
  --nat "any"		NAT port mapping mechanism (any|none|upnp|pmp|extip:<IP>)
  --nodiscover		Disables the peer discovery mechanism (manual peer addition)
  --nodekey 		P2P node key file
  --nodekeyhex 		P2P node key as hex (for testing)

MINER OPTIONS:
  --mine			Enable mining
  --minerthreads "8"		Number of CPU threads to use for mining
  --minergpus 			List of GPUs to use for mining (e.g. '0,1' will use the first two GPUs found)
  --autodag			Enable automatic DAG pregeneration
  --etherbase "0"		Public address for block mining rewards (default = first account created)
  --gasprice "50000000000"	Minimal gas price to accept for mining a transactions
  --extradata 			Block extra data set by the miner (default = client version)

GAS PRICE ORACLE OPTIONS:
  --gpomin "50000000000"	Minimum suggested gas price
  --gpomax "500000000000"	Maximum suggested gas price
  --gpofull "80"		Full block threshold for gas price calculation (%)
  --gpobasedown "10"		Suggested gas price base step down ratio (1/1000)
  --gpobaseup "100"		Suggested gas price base step up ratio (1/1000)
  --gpobasecf "110"		Suggested gas price base correction factor (%)

VIRTUAL MACHINE OPTIONS:
  --vmdebug		Virtual Machine debug output
  --jitvm		Enable the JIT VM
  --forcejit		Force the JIT VM to take precedence
  --jitcache "64"	Amount of cached JIT VM programs

LOGGING AND DEBUGGING OPTIONS:
  --metrics			Enable metrics collection and reporting
  --verbosity "3"		Logging verbosity: 0=silent, 1=error, 2=warn, 3=info, 4=core, 5=debug, 6=detail
  --vmodule 			Per-module verbosity: comma-separated list of <pattern>=<level> (e.g. eth/*=6,p2p=5)
  --backtrace ":0"		Request a stack trace at a specific logging statement (e.g. "block.go:271")
  --pprof			Enable the pprof HTTP server
  --pprofport "6060"		pprof HTTP server listening port
  --memprofilerate "0"		Turn on memory profiling with the given rate
  --blockprofilerate "0"	Turn on block profiling with the given rate
  --cpuprofile 			Write CPU profile to the given file
  --trace 			Write execution trace to the given file

EXPERIMENTAL OPTIONS:
  --shh		Enable Whisper
  --natspec	Enable NatSpec confirmation notice

MISCELLANEOUS OPTIONS:
  --solc "solc"	Solidity compiler command to be used
  --help, -h	show help
```

**Comment by arkpar** [comment]
```
$ geth help account
account command [arguments...]


Manage accounts lets you create new accounts, list all existing accounts,
import a private key into a new account.

'            help' shows a list of subcommands or help for one subcommand.

It supports interactive mode, when you are prompted for password as well as
non-interactive mode where passwords are supplied via a given password file.
Non-interactive mode is only meant for scripted use on test networks or known
safe environments.

Make sure you remember the password you gave when creating a new account (with
either new or import). Without it you are not able to unlock your account.

Note that exporting your key in unencrypted format is NOT supported.

Keys are stored under <DATADIR>/keys.
It is safe to transfer the entire directory or the individual keys therein
between ethereum nodes by simply copying.
Make sure you backup your keys regularly.

In order to use your account to send transactions, you need to unlock them using
the '--unlock' option. The argument is a space separated list of addresses or
indexes. If used non-interactively with a passwordfile, the file should contain
the respective passwords one per line. If you unlock n accounts and the password
file contains less than n entries, then the last password is meant to apply to
all remaining accounts.

And finally. DO NOT FORGET YOUR PASSWORD.


SUBCOMMANDS:
	list	print account addresses
	new	create a new account
	update	update an existing account
	import	import a private key into a new account

```

**Comment by gavofyork** [comment]
Fixed in #666
